src/components/page-private-image-hubs/overview.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const overview = props => (
+  <div className="private-image-hubs-overview">
```

Comment:
Try `className="o-layout-columns"` instead. It's a reusable layout we use in several places already. It's also responsive 🎉